query-string/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): { [key: string]: string | string[] };
+    export function parse(str: string): { [key: string]: string | string[] | null };
```

Comment:
For `parse()` I was wrong! A `| null` is not needed for using `parse()` and check the return value for having key/value pairs with `null`values.

I recall this PR. Sorry!

P.S.: A `| null` is only neccessary for the implementation of `parse()` if it would be written using TypeScript with strict null checks. But that's not the case.